components/tikv_util/src/metrics/mod.rs

```diff
@@ -102,6 +103,8 @@ lazy_static! {
         NON_TXN_COMMAND_THROTTLE_TIME_COUNTER_VEC,
         NonTxnCommandThrottleTimeCounterVec
     );
+    pub static ref INSTANCE_BACKEND_CPU_QUOTA: IntGauge =
+        register_int_gauge!("tikv_backend_cpu_quota", "cpu quota for backend request").unwrap();
```

Comment:
```suggestion
    pub static ref INSTANCE_BACKEND_CPU_QUOTA: IntGauge = register_int_gauge(
        "tikv_backend_cpu_quota",
        "cpu quota for backend request"
    )
    .unwrap();
```

Reply:
> 

make format doesn't permit the change :)

Reply:
That's incorrect. `cargo fmt` doesn't manage macro. So you have to format it yourself.

Reply:
<img width="978" alt="image" src="https://user-images.githubusercontent.com/97348524/175223598-6a40fbaa-3295-4b63-bcb8-4c5ff8e8e144.png">

Reply:
make format will correct my re-format.

Reply:
Okay. How about the other one (`debug!...)`. I tested locally too.